libraries/ui-library/src/components/six-select/six-select.tsx

```diff
@@ -643,21 +704,41 @@ export class SixSelect {
             />
           </div>
 
+          {/* Selection container */}
+          <div
+            class={{
+              'selection-container': true,
+              'selection-container--border': this.selectionContainerItems.length > 0,
```

Comment:
There should not be a border when all items are selected:

<img width="340" alt="Screenshot 2024-01-23 at 17 00 42" src="https://github.com/six-group/six-webcomponents/assets/684943/311d394e-4a29-4987-bedc-0b0a7663c0c8">